scripts/build_erddap_catalog.py

```diff
@@ -570,16 +570,19 @@ def build_erddap_catalog_chunk(data_root, deployment):
 
         qartod_var_type = check_for_qartod_vars(ds)
 
-        exclude_vars = (existing_varnames |
-                        {'latitude', 'longitude'} |
-                        qartod_var_type['qartod'].keys())
+        exclude_vars = (existing_varnames | {'latitude', 'longitude'})
 
         all_other_vars = [add_erddap_var_elem(var) for var in
                           ds.get_variables_by_attributes(name=lambda n: n not in exclude_vars)]
 
         gts_ingest = getattr(ds, 'gts_ingest', 'true')  # Set default value to true
 
-        qartod_vars_snippet = qartod_var_snippets(required_qartod_vars, qartod_var_type)
+        # Exclude automatic QARTOD variables if the dataset is delayed mode
+        # This will keep any user supplied QARTOD variables, they should be part of
+        # `all_other_vars` already
+        qartod_vars_snippet = (qartod_var_snippets(required_qartod_vars,
```

Comment:
the qartod_var_snippets function will continue adding the required QARTOD variables if not in qartod_var_type

Reply:
Yeah, this will need further review, but at least this gets rid of some of the `qartod` default variables for delayed mode datasets.